std/bigint.d

```diff
@@ -1459,10 +1459,7 @@ private:
     {
         return data.isZero();
     }
-    bool isNegative() pure const nothrow @nogc @safe
-    {
-        return sign;
-    }
+    alias isNegative = sign;
```

Comment:
Does this make sign writable? If so, is this desirable?

Reply:
Both `sign` and `isNegative` are private so it should be fine.